app/javascript/mastodon/features/account_timeline/components/familiar_followers.tsx

```diff
@@ -61,23 +62,38 @@ const FamiliarFollowersReadout: React.FC<{ familiarFollowers: Account[] }> = ({
   }
 };
 
-export const FamiliarFollowers: React.FC<{ accountId: string }> = ({
+export const useFetchFamiliarFollowers = ({
```

Comment:
I do not feel good about having hooks like that exported from component files, except if they are really tightly coupled with the component and must be used alongside it

Reply:
Fair point. Would it make more sense in a new `/features/account_timeline/hooks` folder?

It seems too specific for `/mastodon/hooks`.

Reply:
Yes, I think that would be better